nuxeo-compound-documents-web/web-test-runner.config.js

```diff
@@ -0,0 +1,70 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const { createSauceLabsLauncher } = require('@web/test-runner-saucelabs');
+const { legacyPlugin } = require('@web/dev-server-legacy');
+const { playwrightLauncher } = require('@web/test-runner-playwright');
+
+const baseConfig = {
+  files: 'test/**/*.test.js',
+  browsers: [
+    playwrightLauncher({ product: 'chromium' }),
+    // playwrightLauncher({ product: 'webkit' }),
+    // playwrightLauncher({ product: 'firefox' }),
```

Comment:
It is still up to discussion, because to run the tests on those browsers we'll need to update the POD image. Wdyt @Gabez0r? We don't have any requirements about supported browsers, right?

Reply:
Ideally we would test with them, to in line with what we do on Web UI

Reply:
I'm ok with that as well. To do so, I can sync with @ataillefer to check what do we need to install in the container image.

Reply:
OK, so, we need to have a specific builder including all required browsers, currently it's only Chrome.
For Firefox, what minimum version do we need?
For Webkit, what do we need to install?

Reply:
For firefox, I think we should have the latest release, or the last active esr version. 
For Webkit, I don't really know, I believe it is using Chromium, right? I'll confirm with Gabe before you start looking into the issue.